### ISSUE TITLE
Why step so low?

### ISSUE THREAD

**Issue by melontini** [RupertBenWiser/Web-Environment-Integrity#88]
When there are so many more steps above!

This is a great proposal, but it misses the mark in some regards to user safety and privacy.

I recommend going the [Microsoft Defender SmartScreen](https://learn.microsoft.com/en-us/windows/security/operating-system-security/virus-and-threat-protection/microsoft-defender-smartscreen/) route. Why let the user access websites you don't make money from? Right?

If the website's owner decides not to pay you $300+ a year, just block it with the Safe Browsing prompt! Most users won't have any idea anyway, and the owner would be forced to pay you!

Another great idea is to block websites that don't use AdSense! Who knows what types of horrid ads those websites could display?! Google's algorithms would never let any malicious ads through!

And if that's not enough, Chrome is a browser by **Google**, so why even let data points use anything outside of Google's constantly shrinking ecosystem?
 
TLDR: this "spec" is the worst idea ever. Please, stop using Google's products immediately.
